modules/gapi/CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ file(GLOB gapi_ext_hdrs
     "${CMAKE_CURRENT_LIST_DIR}/include/opencv2/${name}/streaming/gstreamer/*.hpp"
     "${CMAKE_CURRENT_LIST_DIR}/include/opencv2/${name}/streaming/onevpl/*.hpp"
     "${CMAKE_CURRENT_LIST_DIR}/include/opencv2/${name}/plaidml/*.hpp"
+    "${CMAKE_CURRENT_LIST_DIR}/include/opencv2/${name}/ported_algorithms/ot/*.hpp"
```

Comment:
Please don't name folders like that.

Note that public include headers is part of the public API. It is not user's concern if the OT algorithm is "ported" or not.

OT's domain is streaming, so I'd assume users to include `opencv2/gapi/streaming/ot.hpp` (or may be `/ot/vasot.hpp`, if multiple OT flavors are possible).

Reply:
Thanks a lot, fixed!